### PR TITLE
Switch to stable Debian, and quash a couple Docker build warnings

### PR DESCRIPTION
**Pull request by mileswwatkins**
With the switch this seems to be running pretty consistently for me on my local machine. Again, stomping all over your turf by touching the Debian install, so let me know if this is misguided :P

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # VERSION   0.1
-FROM        debian:unstable
+FROM        debian:stable
```
Comment (tianon) [inline Dockerfile]: Shouldn't this explicitly be `jessie` for more stability, especially when we get closer to `stretch`?

Comment (mileswwatkins) [inline Dockerfile]: Thanks for the insight!

Comment (tianon) [inline Dockerfile]: Sure, happy to help! :+1:

This is one of those little things that helps smooth out release
transitions.  The best description I ever heard for the "stable" alias was
after I called it a "rolling release" and was told that sure, it's a
rolling release, but it's a very square wheel, so each roll might be a bit
painful. :smile:

**Comment by paultag** [comment]
Seems legit. I might change it to testing at some point in the future if we need it, but this is great! Thanks, @mileswwatkins

**Pull request merged**